README.md

```diff
@@ -1,3 +1,36 @@
+# Cyber City
+
+**Project members:**  
+- Jennifer Telisczak  
+- Brandon Dang  
+- Boo Park  
+- Dante Terminella
+
+**Date:**  
+November 16, 2024
+
+**Course:**  
+2024, Fall, CYBER W295, Section 2 (Wednesday)
```

Comment:
Is there supposed to be a W in front of 295?

Reply:
I was copying it straight from the email. I was wondering if it meant Wednesday.